packages/store-ui/.storybook/decorators/GatsbyIntlProvider.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { IntlProvider } from '@vtex/gatsby-plugin-i18n'
+import MessagesPt from '../i18n/pt.json'
+
+// We can use the code below if we add more languages in the future.
+
+// const locales = ['pt']
+
+// const messages: any = locales.reduce((acc, locale) => {
+//   return {
+//     ...acc,
+//     [locale]: require(`../i18n/${locale}.json`),
+//   }
+// }, {})
+
+const Provider = (Story) => {
+  return (
+    <IntlProvider locale="pt" defaultLocale="pt" messages={MessagesPt}>
```

Comment:
why not `en` as the default ?

Reply:
Because I got this JSON config from Marin haha. 

But I can translate, I think it's better to be in english. Gonna change. ➕